### PR TITLE
added comments to recipes

### PR DESCRIPTION
**Pull request by arya-hemanshu**
added documentation to all the recipes

### CODE REVIEW

File: Samples/aggregate-traffic-count-data-within-localauthorities.py [hunk 1]
```diff
@@ -1,22 +1,38 @@
+'''
+  This recipe gets two datasources
+     - London Traffic data using TrafficCountImporter.java only for London
+     - London localAuthority data using OaImporter.java
+  
+  Then for each localAuthority using GeographicAggregationField
+    - It sums all the bi-cycle count based on LocalAuthority
```
Comment (thanosbnt) [inline Samples/aggregate-traffic-count-data-within-localauthorities.py]: bicycle?

File: Samples/aggregate-traffic-count-data-within-localauthorities.py [hunk 2]
```diff
@@ -1,22 +1,38 @@
...
+  
+  Then it writes the sum to the output file for every localAuthority using GeographicAggregationField
```
Comment (thanosbnt) [inline Samples/aggregate-traffic-count-data-within-localauthorities.py]: Maybe rephrase this to just 'it then sums the individual bicycle counts that are contained within a local aurhority using GeographicAggregationField'

File: Samples/aggregate-traffic-count-data-within-localauthorities.py [hunk 3]
```diff
@@ -1,22 +1,38 @@
...
+'''
 from os import path, pardir
 import sys
 sys.path.append(path.join(path.dirname(path.realpath(__file__)), pardir))
 
 from recipe import Dataset, Subject, AttributeMatcher, GeographicAggregationField, LatestValueField, Match_Rule, Datasource, Recipe
```
Comment (thanosbnt) [inline Samples/aggregate-traffic-count-data-within-localauthorities.py]: say sth like importing all the relevant objects that are needed

File: Samples/aggregate-traffic-count-data-within-localauthorities.py [hunk 4]
```diff
@@ -1,22 +1,38 @@
...
 
+# Creating match rule for London
```
Comment (thanosbnt) [inline Samples/aggregate-traffic-count-data-within-localauthorities.py]: Maybe quickly explain what the attributes mean here

File: Samples/aggregate-traffic-count-data-within-localauthorities.py [hunk 5]
```diff
@@ -1,22 +1,38 @@
...
 datasource_1 = Datasource(importer_class='uk.org.tombolo.importer.ons.OaImporter', datasource_id='localAuthority')
 datasource_2 = Datasource(importer_class='uk.org.tombolo.importer.dft.TrafficCountImporter', 
                         datasource_id= 'trafficCounts', geography_scope=["London"])
 
+# Creating Attribute matcher, which means getting only those values from database where 
+# attribute name is CountPedalCycles
 attribute_matcher = AttributeMatcher(provider='uk.gov.dft', label='CountPedalCycles')
 field = LatestValueField(attribute_matcher=attribute_matcher, label='CountPedalCycles')
 
+# Creating Subject for Geographic aggregation field
```
Comment (thanosbnt) [inline Samples/aggregate-traffic-count-data-within-localauthorities.py]: Mybe explain why we need this?

File: Samples/london-cycle-traffic-air-quality-lsoa-backoff.py [hunk 6]
```diff
@@ -1,34 +1,60 @@
...
+  This recipe gets two datasources
+     - London Traffic data using TrafficCountImporter.java
+     - London localAuthority data using OaImporter.java
+  
+  By using ArithmeticField and LatestValueField if perform two operations
+    - Sum of the mean of 'CarCountTaxis' - 1
+    - Sum of the mean of 'CountPedalCycles' - 2
+  
+  Then for each localAuthority using GeographicAggregationField
+    - It dividies 2/1 to get 'BicycleFraction' - 3
```
Comment (thanosbnt) [inline Samples/london-cycle-traffic-air-quality-lsoa-backoff.py]: Divides 2/1 = 2? :) Maybe we need to describe this in words

File: Samples/london-cycle-traffic-air-quality-lsoa-backoff.py [hunk 7]
```diff
@@ -1,34 +1,60 @@
...
+  
+  Then writes 3 to the output file for every localAuthority using GeographicAggregationField
```
Comment (thanosbnt) [inline Samples/london-cycle-traffic-air-quality-lsoa-backoff.py]: Same as before: we can say it then sums the resulting field...

**Review by thanosbnt — CHANGES_REQUESTED** [review]
Generally good, i think we need to be more descriptive in the comments...We can do it tohether if you want

File: Samples/aggregate-traffic-count-data-within-localauthorities.py (hunk 2 above)
Comment (arya-hemanshu) [reply Samples/aggregate-traffic-count-data-within-localauthorities.py]: done

File: Samples/aggregate-traffic-count-data-within-localauthorities.py (hunk 3 above)
Comment (arya-hemanshu) [reply Samples/aggregate-traffic-count-data-within-localauthorities.py]: done

File: Samples/aggregate-traffic-count-data-within-localauthorities.py (hunk 1 above)
Comment (arya-hemanshu) [reply Samples/aggregate-traffic-count-data-within-localauthorities.py]: done

File: Samples/aggregate-traffic-count-data-within-localauthorities.py (hunk 4 above)
Comment (arya-hemanshu) [reply Samples/aggregate-traffic-count-data-within-localauthorities.py]: done

File: Samples/aggregate-traffic-count-data-within-localauthorities.py (hunk 5 above)
Comment (arya-hemanshu) [reply Samples/aggregate-traffic-count-data-within-localauthorities.py]: done

File: Samples/london-cycle-traffic-air-quality-lsoa-backoff.py (hunk 6 above)
Comment (arya-hemanshu) [reply Samples/london-cycle-traffic-air-quality-lsoa-backoff.py]: done

File: Samples/london-cycle-traffic-air-quality-lsoa-backoff.py (hunk 7 above)
Comment (arya-hemanshu) [reply Samples/london-cycle-traffic-air-quality-lsoa-backoff.py]: done

**Comment by arya-hemanshu** [comment]
@thanosbnt made the suggested changes, could you please have a second round of review :)

**Review by thanosbnt — APPROVED** [review]
